app/assets/stylesheets/application.css

```diff
@@ -129,3 +129,15 @@ label + input {
   border-radius: 5px;
   text-align: left;
 }
+
+.well_done {
```

Comment:
If we have time we should revisit the names of some of these... its a work in progress guys, sorry I'm so bad at names!

Reply:
hardest part of programming... we can make it part of the styling refactor push...